zero_functional.nim

```diff
@@ -39,8 +42,8 @@ else:
     const defaultCollectionType = "seq"
   const defaultResultType = defaultCollectionType & "[int]"
 
-proc print_code(code: NimNode) =
-  echo(repr(code).replace("`gensym", "_").replace("__call__", "Call").replace("__", ""))
+proc printCode(code: NimNode) =
+  echo(repr(code).replace("`gensym", "_")#[.replace(callSuffix, "Call")]#.replace("_", ""))
```

Comment:
this should be restored - see comment above concerning the internal variable names